tools/README.md

```diff
@@ -27,131 +31,194 @@ Change as necessary...
 Fill in this documentation about parameters and labels from pipeline-cli...
```

Comment:
we should remove all of this if we are not using it. it's more confusing than helpful if we don't use those variables/parameters.

Reply:
Done